pom.xml

```diff
@@ -133,6 +133,12 @@
             <version>2.2</version>
         </dependency>
 
+		<dependency>
+           <groupId>org.owasp.encoder</groupId>
```

Comment:
Doesn't having the same library in multiple modules result into this problem? https://issues.openmrs.org/browse/TRUNK-5344

How about just relying on the one in core? https://github.com/openmrs/openmrs-core/blob/2.0.0/web/pom.xml#L116-L120

Reply:
I have changed the implementation to use the escapeHTML() from the already existing WebUtil class from the core.